gridded_etl_tools/utils/zarr_methods.py

```diff
@@ -1126,3 +1169,10 @@ def are_times_in_expected_order(self, times: tuple[datetime.datetime], expected_
             previous_time = instant
         # Return True if no problems found
         return True
+
+    def post_parse_quality_check(self):
```

Comment:
I think we want to this to run in Prefect where the possibility of an auto revert or a change to API retrieval could happen. Currently this repo doesn't have any knowledge of Prefect or the API and I think we want to keep it that way.

Reply:
Yeah agreed the more I thought about this the more it belongs as a standalone script we run in Prefect. I’ll remove this function and we can implement it elsewhere